bin/raptor-regression

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+var raptorRegression = require('..');
+var userArgs = process.argv;
+
+if (userArgs.length < 4) {
+  console.log("USAGE: %s data.json app.context <app.entrypoint>", userArgs[0]);
+  process.exit(1);
+}
+
+var dataFile = userArgs[2];
+var appContext = userArgs[3];
+if (userArgs.length == 5) {
+  var appEntrypoint = userArgs[4];  
```

Comment:
Although, you could probably just change this to not have a variable at all:

```js
raptorRegression(dataFile, appContext, userArgs[4]);
```

Then just look for falsey values at index.js line 38.